cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/RunCommand.java

```diff
@@ -114,8 +115,13 @@ public void execute() {
         String[] args;
         if (this.argList == null) {
             args = new String[0];
+            this.projectPath = Paths.get(System.getProperty("user.dir"));
```

Comment:
Can't we use `USER_DIR` in ProjectConstants?

Reply:
Fixed in 96cd790